app/(request)/private-cloud/edit/[licencePlate]/page.tsx

```diff
@@ -93,6 +93,18 @@ export default function EditProject({ params }: { params: { licencePlate: string
     }
   }, [requestData]);
 
+  useEffect(() => {
+    if (data?.secondaryTechnicalLead) {
+      setSecondTechLead(true);
+    }
+  }, [data]);
+
+  useEffect(() => {
```

Comment:
is this a `duplication` of above?

Reply:
Yes but they are in different pages. So we want the secondary technical lead to be expanded in each of those pages if it exists